src/HonzaBotner.Database/Verification.cs

```diff
@@ -7,5 +7,7 @@ public class Verification
     [Key] public ulong UserId { get; set; }
     public string AuthId { get; set; }
 
+    public string Username { get; set; }
+
     public override string ToString() => $"Discord: <@!{UserId}>, auth hash: {AuthId}";
```

Comment:
Should be updated with `Username`.

Reply:
Very temporary, we just need to start saving usernames asap, before freshmen start piling up, so that when the real changes come, we don't have to reverify that many people. There will be a major rewrite, just not yet.